istioctl/cmd/analyze.go

```diff
@@ -123,6 +130,8 @@ istioctl experimental analyze -k a.yaml b.yaml
 
 	analysisCmd.PersistentFlags().BoolVarP(&useKube, "use-kube", "k", false,
 		"Use live kubernetes cluster for analysis")
+	analysisCmd.PersistentFlags().StringVarP(&useDiscovery, "discovery", "d", "",
```

Comment:
Add an example of this to the `Example:` for the command.  I am not sure how to use it.

Reply:
Added a couple of examples.